lib/engine/db.go

```diff
@@ -164,3 +144,10 @@ func insert(db *mgo.Database) chan *Value {
 
 	return source
 }
+
+// FlushImportedData finalise l'insertion des données dans ImportedData.
+func FlushImportedData(channel chan *Value) {
+	channel <- &Value{}
```

Comment:
Je trouve que l'envoi d'un objet vide puis la fermeture du channel sont un peu redondants.
L'envoi d'une valeur vide était une façon d'envoyer un message à la routine d'insert pour lui demander d'envoyer les derniers objets de son cache dans la base même si elle n'avait pas atteint les 100 objets, mais c'était dans un contexte ou la routine ne s'arrétait jamais.
Ne pourrait-on pas plutôt faire en sorte que cela soit fait lorsque le channel est fermé ? Je me souviens qu'il y a dans cette routine une condition qui test si l'objet est vide ou si la condition de 100 objets est réunie pour faire l'insert, du coup on pourrait retirer cette condition d'objet vide, ne plus l'envoyer, et faire l'insert final après le for ? Ici du coup il suffira de fermer le channel pour provoquer la fin.
Dans tous les cas c'est pas du tout bloquant pour valider la PR, ça va fonctionner :)

Reply:
Très bonne idée ! Comme tu le suggérais en daily, je vais fusionner tel quel puis apporter cette amélioration dans une PR séparée. Merci Christophe !

Reply:
C'est fait, dans https://github.com/signaux-faibles/opensignauxfaibles/pull/269/files/702aecde28ba6905d71c008fa489c0335849dde9#diff-2dc69192073f76b771bb9d03273632fda1b8fa6364c4e750b70159c8549426e2